src/plugins/plugin.legend.js

```diff
@@ -659,21 +659,19 @@ export class Legend extends Element {
 		const hoveredItem = me._getLegendItemAt(e.x, e.y);
 
 		if (type === 'click') {
-			if (hoveredItem && opts.onClick) {
-				// use e.native for backwards compatibility
-				opts.onClick.call(me, e.native, hoveredItem);
+			if (hoveredItem) {
+				call(opts.onClick, [e, hoveredItem, me]);
```

Comment:
Can we pass `me` as the third arg? Then I think `this` will still work for normal functions so that this isn't a breaking change

Reply:
Done. Good idea.